ports/thecorruptionwithin/The Corruption Within.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+XDG_DATA_HOME=${XDG_DATA_HOME:-$HOME/.local/share}
+
+if [ -d "/opt/system/Tools/PortMaster/" ]; then
+  controlfolder="/opt/system/Tools/PortMaster"
+elif [ -d "/opt/tools/PortMaster/" ]; then
+  controlfolder="/opt/tools/PortMaster"
+elif [ -d "$XDG_DATA_HOME/PortMaster/" ]; then
+  controlfolder="$XDG_DATA_HOME/PortMaster"
+else
+  controlfolder="/roms/ports/PortMaster"
+fi
+
+source $controlfolder/control.txt
+[ -f "${controlfolder}/mod_${CFW_NAME}.txt" ] && source "${controlfolder}/mod_${CFW_NAME}.txt"
+get_controls
+
+# Variables
+GAMEDIR="/$directory/ports/thecorruptionwithin"
+
+# CD and set permissions
+cd $GAMEDIR
+> "$GAMEDIR/log.txt" && exec > >(tee "$GAMEDIR/log.txt") 2>&1
+$ESUDO chmod +x -R $GAMEDIR/*
```

Comment:
Only two files need the execution flag :)
```suggestion
$ESUDO chmod +x $GAMEDIR/gmloadernext.aarch64
$ESUDO chmod +x $GAMEDIR/tools/splash
```

Reply:
I don't think we should be overly nitpicky with the permissions and armeabi libs included. We haven't fashioned any new standards to follow here so it will confuse engineers. Also a single `$ESUDO chmod +x -R $GAMEDIR/*` is cleaner than setting permissions for individual files, and remember this line only exists because people will install ports manually.

Reply:
@tabreturn  you are free to choice how you want to do it, see this change request more as a suggestion.